src/ngraph/frontend/onnx_import/op/conv_transpose.cpp

```diff
@@ -171,23 +171,27 @@ namespace ngraph
                         }
                         for (int i = 0; i < num_spatial_dims; ++i)
                         {
-                            padding_below[i] = (strides[i] * (data_shape[i + 2] - 1) +
-                                                dilations[i] * (weights_shape[i + 2] - 1) -
-                                                data_dilation_strides[i] *
-                                                    (output_shape[i] - output_padding[i] - 1)) /
-                                               2;
-                            if (static_cast<int>(padding_below[i]) < 0)
+                            padding_below[i] = strides[i] * (data_shape[i + 2] - 1) +
+                                               dilations[i] * (weights_shape[i + 2] - 1) -
+                                               data_dilation_strides[i] *
+                                                   (output_shape[i] - output_padding[i] - 1);
+                            if (padding_below[i] < 0)
                             {
-                                output_padding[i] = -static_cast<int>(padding_below[i]);
-                                padding_below[i] = 0;
+                                // (int) -9 / 2 = -5 but we need -4
+                                // (int) -9 --> 9 / 2 = 4 --> -4
+                                padding_below[i] = -(-padding_below[i] / 2);
+                            }
+                            else
+                            {
+                                padding_below[i] /= 2;
```

Comment:
Add a comment why we need to divide this value by 2

Reply:
to get symmetric above and below padding.